Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -221,9 +221,10 @@ - (void)loadView
     UILabel *emptyBoxLabel = [UILabel new];
     self.emptyBoxLabel = emptyBoxLabel;
     [self.view addSubview:emptyBoxLabel];
-    [emptyBoxLabel autoPinWidthToSuperview];
-    [emptyBoxLabel autoPinToTopLayoutGuideOfViewController:self withInset:0];
-    [emptyBoxLabel autoPinToBottomLayoutGuideOfViewController:self withInset:0];
+    [emptyBoxLabel setNumberOfLines:0]; //  Let the label use as many lines as needed. It will very rarely be more than 2.
```

Comment:
style nit: no trailing comments, leave them above on a line of their own

Reply:
No problem, learning the local coding style as we go.